cypress/integration/search/results_post_comment_spec.js

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 3]*/
+
+describe('Search', () => {
+    it('S14548 Search results Right-Hand-Side: Post a comment', () => {
+        // 1. Login and navigate to the app
+        cy.login('user-1');
+        cy.visit('/');
+
+        const message = `asparagus-${Date.now()}`;
+        const comment = 'Replying to asparagus';
+
+        // 2. Post a new message
+        cy.postMessage(message);
+
+        // 3. Search for the text we just entered
+        cy.get('#searchBox').type(message).type('{enter}');
+
+        // 4. Get last postId
+        cy.getLastPostId().then((postId) => {
+            const postMessageText = `#postMessageText_${postId}`;
+
+            // * Search results should have our original message
+            cy.get('#search-items-container').find(postMessageText).should('have.text', `${message}\n`);
+
+            // 5. Click on the reply button on the search result
+            // This button is visible only on hover. As a result,
+            // we have to force click here because we cannot
+            // trigger a CSS hover state.
+            // See - https://github.com/cypress-io/cypress/issues/1485#issuecomment-386085145
+            cy.get(`#SEARCH_commentIcon_${postId}`).click({force: true});
```

Comment:
```suggestion
            cy.clickPostCommentIcon(postId, 'SEARCH');
```

Please use the command we have for it.  I agree on the comment but I guess we could omit it.

Reply:
@saturninoabril Bah! Missed that command! All fixed up now.

Reply:
No worries.  We're just starting out build all those commands and might need to reorganize an better document in-code in the future.